experiments/model_development/README.md

```diff
@@ -23,7 +44,7 @@ To run a sample training job, you need to
                     iteration_data.pickle
             ⋮
     ```
-    Alternatively, you may specify a path to a folder following the previous format in `config/config.yaml`. A toy example of a data directory is available [here](https://www.dropbox.com/s/50j2mi9kqls3v7k/data.zip).
```

Comment:
the prior link to the toy dataset seems to be broken: dropbox says that the file may have been deleted